### PR TITLE
#17 Score is calculated and displayed on screen.

### LINKED ISSUE #17: Increasing score
Score should be based on distance run + amount of coins collected

### PR DESCRIPTION
**Pull request by mdingjan**
Will close #17

Please review :)

### CODE REVIEW

**Comment by MathiasMeuleman** [comment]
I merged the master in for you. Please fix the checkstyle in State.java, then you are done I think

File: src/main/java/nl/tudelft/ti2206/group9/level/Track.java [hunk 1]
```diff
@@ -193,6 +193,7 @@ public final synchronized void step() {
 		getPlayer().step();
 
 		distance += UNITS_PER_TICK;
+		State.addScore(UNITS_PER_TICK);
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/level/Track.java]: Maybe we should still write a nice formula for this, but for now it's good enough :)

**Comment by mpsijm** [comment]
Review finished, apart from checkstyle warnings it's all good :D

**Comment by mpsijm** [comment]
Thanks for fixing, will merge :D

**Pull request merged**